test/run-stp-tests.sh

```diff
@@ -1025,7 +1029,15 @@ else
 fi
 
 # Error log messages
-count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup' | wc -l | awk '{print $1}'`
+count=`$GREP '^E ' $BASE/log/error_log | $GREP -v 'Unknown default SystemGroup' | \
+       $GREP -v '(usb) crashed on signal 11' | \
```

Comment:
Um, that sounds like it should be an issue that is investigated...

Reply:
Indeed. I'll remove these two and report back.

Reply:
https://buildd.debian.org/status/package.php?p=cups&suite=experimental

I dropped this exception and rebuilt; no issue.